app.js

```diff
@@ -1,4 +1,33 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+'use strict';
+var express = require('express');
+var trucks = require('./trucks');
+var app = express();
+
+app.use(express.static('public'));
+app.get('/', function (request, response) {
```

Comment:
This route is unnecessary as the static middleware has already handled the root path by the time it gets to this handler.